File: root/cgi-bin/time.py
import datetime
import sys

def main():
    current_time = datetime.datetime.now()

    formatted_time = current_time.strftime("%A, %B %d, %Y %I:%M:%S %p")

    print("HTTP/1.1 200 OK")
    print("Content-Type: text/html")
    print("Access-Control-Allow-Origin: *")
    print("Access-Control-Allow-Methods: GET, POST, DELETE")
    print("Access-Control-Allow-Headers: *")
    print()

    html_content = f"""
    <html>
        <head>
            <title>Current Time</title>
            <style>
                body {{
                    font-family: Arial, sans-serif;
                    background-color: #f0f8ff;
                    text-align: center;
                    padding: 50px;
                }}
                h1 {{
                    color: #333;
                    font-size: 48px;
                    margin-bottom: 20px;
                }}
                .time {{
                    font-size: 36px;
                    font-weight: bold;
                    color: #4CAF50;
                }}
                .footer {{
                    margin-top: 50px;
                    font-size: 18px;
                    color: #666;
                }}
            </style>
        </head>
        <body>
            <h1>Current Time</h1>
            <div class="time">
                {formatted_time}
            </div>
            <div class="footer">
                <p>Powered by Python</p>
            </div>
        </body>
    </html>
    """

    print(html_content)
    

if __name__ == "__main__":
    main()

File: handin/1/root/cgi-bin/time.php
<?php
$current_time = date("l, F d, Y h:i:s A");

header("Content-Type: text/html");
header_remove("Content-Type");

$html_content = "<html>";
$html_content .= "<head>";
$html_content .= "<title>Current Time</title>";
$html_content .= "<style>";
$html_content .= "body {";
$html_content .= "font-family: Arial, sans-serif;";
$html_content .= "background-color: #f0f8ff;";
$html_content .= "text-align: center;";
$html_content .= "padding: 50px;";
$html_content .= "}";
$html_content .= "h1 {";
$html_content .= "color: #333;";
$html_content .= "font-size: 48px;";
$html_content .= "margin-bottom: 20px;";
$html_content .= "}";
$html_content .= ".time {";
$html_content .= "font-size: 36px;";
$html_content .= "font-weight: bold;";
$html_content .= "color: #4CAF50;";
$html_content .= "}";
$html_content .= ".footer {";
$html_content .= "margin-top: 50px;";
$html_content .= "font-size: 18px;";
$html_content .= "color: #666;";
$html_content .= "}";
$html_content .= "</style>";
$html_content .= "</head>";
$html_content .= "<body>";
$html_content .= "<h1>Current Time</h1>";
$html_content .= '<div class="time">';
$html_content .= $current_time;
$html_content .= "</div>";
$html_content .= '<div class="footer">';
$html_content .= "<p>Powered by PHP</p>";
$html_content .= "</div>";
$html_content .= "</body>";
$html_content .= "</html>";

echo ("HTTP/1.1 200 OK\n");
echo ("Content-Type: text/html\n");
echo ("Access-Control-Allow-Origin: *\n");
echo ("Access-Control-Allow-Methods: GET, POST, DELETE\n");
echo ("Access-Control-Allow-Headers: *\n");
echo ("Content-Length: " . strlen($html_content) . "\r\n\r\n");

echo $html_content;
?>